presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -41,7 +41,7 @@
         "task.level-absolute-priority"})
 public class TaskManagerConfig
 {
-    private boolean verboseStats;
```

Comment:
I suggest we leave this disabled until we can confirm that we don't still get a regression.

Reply:
@arhimondr do you have any numbers for the overhead of enabling cpu tracking with `verboseStats=true`?

Reply:
done

Reply:
@nezihyigitbasi Here is some data gathered by doing syntetic tests. It supposed to do not introduce any additional overhead. Removing the user cpu tracking must cover the costs of enabling the overall cpu tracking per operator. Anyhow, i'm going to leave it dissabled by default, and run a profiler on the real cluster heavy loaded with a real workload to make sure the CPU tracking code does not show up in the profiler results.